src/HonzaBotner.Discord.Services/Commands/MemberCommands.cs

```diff
@@ -28,8 +26,8 @@ public class MemberCommands : BaseCommandModule
 {
     [Group("info")]
     [Aliases("about", "whois")]
-    [OnlyForMods]
     [Description("Provides info about a member.")]
+    [OnlyForMods]
```

Comment:
I aslo don't really understand the rationality for renaming this when we could just use `RequireMod` and `RequireTeacher`. It should be again in line with other attributes, like the ones from lib. Maybe rename it to `RequireModAttribute` and `RequireWhateverYouDecideInTheEndAttribute` instead?
![image](https://user-images.githubusercontent.com/29132060/153499305-aabcf3f6-fa3c-4b7a-b362-44e038de85ba.png)

Reply:
This comment was regarding RequireMod rename only. Previous was about the name of teacher attribute. I don't understand why you've decided to change something that works well to something that is even more... well not standard.

Reply:
If you look at lib-provided attributes, the names are `RequireXAttribute` for attributes that modify availability. We were more or less in line with this with `RequireMod` so the change `OnlyForMods` is completely oposite direction than it should follow. And I am still not touching the fact that this particular change is not needed at all 😄 And **if** you want to change anything here, it should just be `RequireMod` -> `RequireModAttribute`. But `OnlyForMods` is just completely out of the line.